iset.mm

```diff
@@ -129911,11 +129911,16 @@ then the Limited Principle of Omniscience (LPO) implies excluded middle.
     ctiunct.a $e |- ( ph -> F : _om -onto-> ( A |_| 1o ) ) $.
     ctiunct.b $e |- ( ( ph /\ x e. A )
       -> G : _om -onto-> ( B |_| 1o ) ) $.
-    $( The union of countably many countable sets is countable.  The
-       construction here is constructive in the sense that the hypothesis
-       provides a specific class for ` G ` in terms of ` x ` , as opposed to a
-       construction of the form ` A. x e. A E. g ` .  Countable choice would be
-       needed to turn the latter into the former.
+    $( A sequence of enumerations gives an enumeration of the union.  We refer
+       to "sequence of enumerations" rather than "countably many countable
+       sets" because the hypothesis does not provide countability for each
+       ` B ( x ) ` in isolation (independent of how it is enumerated), but in
+       terms of the ` B ( x ) ` together with the ` G ( x ) ` which enumerates
+       it.  Countable choice would be needed to turn the "countably many
+       countable sets" version into the "sequence of enumerations" version.
+
+       Compare with when we have two sets instead of countably many, as seen at
+       ~ unct , in which we can express countability using ` E. ` .
```

Comment:
```suggestion
       Compare with the case of two sets instead of countably many, as seen at
       ~ unct , in which case we can express countability using ` E. ` .
```

Reply:
Suggestion accepted and pull request updated.